GoogleForCreators/web-stories-wp
# Workspace: Support default padding styles for highlight and fill modes.

## Summary

Set a specific padding value when choosing highlight or fill text options.

## Relevant Technical Choices

- Moves the padding calculations to a separate file since it is now used by both the padding and color panes

## To-do

- None

## User-facing changes

- A highlight fill now sets a 10 x 4 padding and a background fill sets a 6 x 6 padding.

## Testing Instructions

1. Open a story in the editor
2. Add a text box
3. Select Highlight text mode and see a 10pt vertical and 4pt horizontal padding set
Or
3. Select Fill text mode and see a 6pt padding set

---



#4972 #4980
